src/sparseml/pytorch/utils/loss.py

```diff
@@ -213,10 +213,36 @@ class CrossEntropyLossWrapper(LossWrapper):
 
     def __init__(
         self,
+        label_smoothing: float = 0.0,
```

Comment:
will this break on `torch<1.10`?

Reply:
To be exact, line `230` (`self._loss_fn`) will throw an error, because on lower pytorch versions the loss function will not recognise the `label_smoothing` argument.

Reply:
let's definitely make sure this is compatible with lower torch versions (ie check torch version and only include label smoothing if >=1.10 and raise an error if torch < 1.10 and label_smoothing != 0)

Reply:
Refactored.